Issue 1524: security-compliance subdir updates (#1531)
raft-tech/TANF-app: docs/Security-Compliance/Secret-Key-Mgmt.md

The goal of these efforts is to help prevent unintended secret key leakage. Though it isn't possible to completely eliminate the possibility of secret key leakage, the strategies documented herein will help improve incident response in the event where keys are leaked. 

_See [secret key leakage post-mortem summary](https://hhsgov.sharepoint.com/sites/TANFDataPortalOFA/_layouts/15/Doc.aspx?sourcedoc={cbce2e75-17b2-4e70-b422-60d034fcd4af}&action=edit&wd=target%28Dev%20Notes.one%7C3dbb7d3a-694d-4f1c-a656-f907991c1f7d%2FSecret%20Key%20Leakage%20Post-Mortem%20Synthesis%7C0496800f-8810-4159-95e4-9fc605dc86d4%2F%29) for more details on how this strategy was informed._  

Herein the following is described:

> - [what secret keys are and how these can be leaked]()

Author: ```suggestion
- [what secret keys are and how these can be leaked](##What-are-secret-keys,-and-how-can-these-be-leaked?)
```

> - [what secret keys are and how these can be leaked](#What-are-secret-keys,-and-how-can-these-be-leaked?)

Author: ```suggestion
- [what secret keys are and how these can be leaked](#What-are-secret-keys-and-how-can-these-be-leaked?)
```

> - [what secret keys are and how these can be leaked](##What-are-secret-keys,-and-how-can-these-be-leaked?)

Author: ```suggestion
- [what secret keys are and how these can be leaked](## What-are-secret-keys,-and-how-can-these-be-leaked?)
```

> - [what secret keys are and how these can be leaked](## What-are-secret-keys,-and-how-can-these-be-leaked?)

Author: ```suggestion
- [what secret keys are and how these can be leaked](#What-are-secret-keys,-and-how-can-these-be-leaked?)
```

> - [steps we are taking to minimize potential for secret key leakage]()

Author: ```suggestion
- [steps we are taking to minimize potential for secret key leakage](#Mitigation-practices)
```

- [incident response protocol if secret keys are leaked or at risk of exposure]()

> ## What are secret keys, and how can these be leaked?

Author: ```suggestion
##What are secret keys, and how can these be leaked?
```

> - [incident response protocol if secret keys are leaked or at risk of exposure]()

Author: ```suggestion
- [incident response protocol if secret keys are leaked or at risk of exposure](#Communication-protocol-if-secret-keys-are-leaked)
```

> ## What are secret keys, and how can these be leaked?

Author: ```suggestion
## What are secret keys and how can these be leaked?
```

> - [what secret keys are and how these can be leaked](#What-are-secret-keys-and-how-can-these-be-leaked?)

Author: ```suggestion
- [what secret keys are and how these can be leaked](#What-are-secret-keys-and-how-can-these-be-leaked)
```

> ##What are secret keys, and how can these be leaked?

Author: ```suggestion
## What are secret keys, and how can these be leaked?
```

> As described [here](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Technical-Documentation/Architecture%20Decision%20Record/004-configuration-by-environment-variable.md), the TDP system is configured by environment variables. [Environment variables](https://medium.com/chingu/an-introduction-to-environment-variables-and-how-to-use-them-f602f66d15fa) help minimize the risk of running into unexpected issues when changes are made to the system. Some of these environment variables are senstive because these are used to authorize access to TDP. We refer to these as "Secret keys". 

Author: ```suggestion
As described [here](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Technical-Documentation/Architecture-Decision-Record/004-configuration-by-environment-variable.md), the TDP system is configured by environment variables. [Environment variables](https://medium.com/chingu/an-introduction-to-environment-variables-and-how-to-use-them-f602f66d15fa) help minimize the risk of running into unexpected issues when changes are made to the system. Some of these environment variables are senstive because these are used to authorize access to TDP. We refer to these as "Secret keys". 
```
